docs/en/user_guides/prepare_datasets.md

```diff
@@ -155,3 +155,67 @@ Here is a processed sample
 ![transformed_coco](https://user-images.githubusercontent.com/26127467/187386652-bd47335d-797c-4e8c-b823-2a4915f9812f.jpg)
 
 The heatmap target will be visualized together if it is generated in the pipeline.
+
+## Download dataset via MIM
+
+By using [OpenDataLab](https://opendatalab.com/), you can obtain free formatted datasets in various fields. Through the search function of the platform, you may address the dataset they look for quickly and easily. Using the formatted datasets from the platform, you can efficiently conduct tasks across datasets.
+
+If you use MIM to download, make sure that the version is greater than v0.3.8. You can use the following command to update:
+
+```shell
+# upgrade your MIM
+pip install -U openmim
+
+# install OpenDataLab CLI tools
+pip install -U opendatalab
+# log in OpenDataLab, registry
+odl login
+
+# download coco2017 and preprocess by MIM
+mim download mmpose --dataset coco2017
+```
+
```

Comment:
Perhaps we could include an example command for extracting data from the downloaded file using the new scripts?

Reply:
The preprocessing script will be automatically executed by mim